packages/block-library/src/shortcode/editor.scss

```diff
@@ -1,25 +1,10 @@
-.wp-block-shortcode {
-	display: flex;
-	flex-direction: column;
-	padding: $block-padding;
-	font-size: $default-font-size;
-	font-family: $default-font;
-	margin-bottom: $default-block-margin;
-
-	label {
-		display: flex;
-		align-items: center;
-		white-space: nowrap;
-		font-weight: 600;
-		flex-shrink: 0;
-	}
-
+[data-type="core/shortcode"] {
```

Comment:
Why the selector change?

Reply:
For whatever reason the previous selector no longer exists. I did not think it important enough to dig into why.